test/geocel/GeoTests.cc

```diff
@@ -403,8 +397,10 @@ void FourLevelsGeoTest::test_detailed_tracking() const
         else if (test_->geometry_type() == "ORANGE")
```

Comment:
If we're changing the direction now, maybe vecgeom will also work? So let's see what happens when you delete the `if` here.